apps/explorer/src/ui/Link.tsx

```diff
@@ -42,14 +44,22 @@ export function Link({
     before,
     after,
     children,
+    gap = 'md',
```

Comment:
Why isn't this just a consistent presentation? Why do we have variable gaps?

Reply:
Default is `gap-2` between content and the Icon. We need it to be smaller to `gap-0.5` for the render of `Object >`. I could pass the content + Icon in as `children` but then it would defeat the purpose of using `before` and `after` here.

Reply:
Updating to `gap-0.5` might not be correct for the use case elsewhere

Reply:
If there's one special-case, then special case it in the usage vs bloating the component with the configuration (ideally we limit the amount of special casing this way by making it hard, not easy). The goal is for you to be annoyed at it and work with design to drive consistency. Or if there is a legit case where we need variants, add it very thoughtfully.